src/api/usersAPI.ts

```diff
@@ -76,3 +76,18 @@ export const checkNickname = async (nickname: string) => {
     throw error;
   }
 };
+
+/**
+ * 클릭한 사용자의 정보를 반환합니다.
+ */
+
+export const getOthersInfo = async (userId: number) => {
+  try {
+    const res = await fetch(`${BASE_URL}/api/v1/users/${userId}`);
+    const result = await res.json();
+    const { data } = result;
```

Comment:
구조 분해로 줄일 수 있을 것 같아요
```suggestion
    const { data }= await res.json();
```